packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -158,6 +158,22 @@ const ruleset: any = {
       },
     },
 
+    // RPC Code: RPC-Async-V1-06
+    LroAzureAsyncOperationHeader: {
+      rpcGuidelineCode: "RPC-Async-V1-06",
+      description: "Azure-AsyncOperation header must be supported for all async operations that return 202.",
+      message: "{{description}}",
+      severity: "error",
+      formats: [oas2],
+      given: "$.paths[*][*].responses[?(@property == '202')]",
```

Comment:
>s[?(@property == '202') [](http://example.com/codeflow?start=36&length=23)

This rule is actually for all long running operations. Not just for the 202 status code. Sorry I should have clarified this earlier.